### ISSUE TITLE
Param of Batchnorm2d

### ISSUE THREAD

**Issue by anmol039w** [pytorch/pytorch#23936]
The number of parameters of Batchnorm2d layer when compared to that of keras is exactly half,

so for an input conv2d layer with 64 filters the parms of Batchnorm2d in Keras is 256 whereas in pytorch is 128, can you please explain the reason behind it

**Comment by soumith** [comment]
please use https://discuss.pytorch.org for questions
